test/libsolidity/SyntaxTest.cpp

```diff
@@ -132,8 +132,13 @@ void SyntaxTest::filterObtainedErrors()
 			// ignore the version & license pragma inserted by the testing tool when calculating locations.
 			if (location->start != -1)
 			{
-				solAssert(location->start >= preambleSize, "");
-				locationStart = location->start - preambleSize;
+				if (location->start >= preambleSize)
+					locationStart = location->start - preambleSize;
+				else
+				{
+					solAssert(location->start == 0, "");
+					locationStart = 0;
+				}
 			}
 			if (location->end != -1)
```

Comment:
Not adjusting `end` will result in the error not covering the whole file. This might look a bit weird, but I don't have a good solution other than making the SMTChecker aware of the preamble and that sounds like bad idea.